utilities/fault_injection_fs.h

```diff
@@ -497,6 +503,13 @@ class FaultInjectionTestFS : public FileSystemWrapper {
   // saved callstack
   void PrintFaultBacktrace();
 
+  void AddThreadLocalMessage(const std::string& m) {
```

Comment:
This is not being called from anywhere. Dead code?

Reply:
Yes I can remove it. I used it for adding manual code in all the places to debug...